.github/files/renovate-post-upgrade.sh

```diff
@@ -41,14 +42,19 @@ fi
 #pnpm config set --global store-dir /tmp/renovate/cache/others/pnpm
 #composer config --global cache-dir /tmp/renovate/cache/others/composer
 
-# Do the pnpm and changelogger installs.
+# Do the pnpm install. Turn off some strictness settings to make it more likely this will work.
 cd "$BASE"
+echo 'strict-peer-dependencies = false' >> .npmrc
+echo 'strict-dep-builds = false' >> .npmrc
 pnpm install
-cd projects/packages/changelogger
+git restore .npmrc
+
+# Install changelogger too.
+cd "$BASE/projects/packages/changelogger"
```

Comment:
Presumably the `$BASE` was added here for consistency rather than being required?